infrastructure/ansible/roles/ats_sim/templates/env.j2

```diff
@@ -0,0 +1,5 @@
+TO_URL={{sim_to_url}}
+TO_CDN={{sim_to_cdn}}
+TO_USER={{sim_to_user}}
+TO_PASSWORD='{{sim_to_password}}'
+SIMULATOR_PORT={{simulator_port}}
```

Comment:
Spaces should surround variables inside interpolations.

Reply:
done